src/lib/ngc/compile-source-files.ts

```diff
@@ -47,8 +47,8 @@ export async function compileSourceFiles(
     rootNames: tsConfig.rootNames,
     options: tsConfigOptions,
     emitCallback: createEmitCallback(tsConfigOptions),
-    host: ngCompilerHost,
-    oldProgram: ngProgram
+    emitFlags: tsConfig.emitFlags,
```

Comment:
I had to re-add this as Angular Compiler are not handling all cases properly.
https://github.com/angular/angular/blob/5.2.x/packages/compiler-cli/src/perform_compile.ts#L152

If I don't add this, `ngsummary` files will be generating when adding `fullTemplateTypeCheck` even though `skipTemplateCodegen` is set

I'll try to have a look at it and open an issue with Angular.

Reply:
https://github.com/angular/angular/issues/23662